homeassistant/components/switch/dlink.py

```diff
@@ -74,37 +78,28 @@ def device_state_attributes(self):
         try:
             ui_temp = self.units.temperature(int(self.data.temperature),
                                              TEMP_CELSIUS)
-            temperature = "%i %s" % \
-                          (ui_temp, self.units.temperature_unit)
+            temperature = ui_temp
         except (ValueError, TypeError):
-            temperature = STATE_UNKNOWN
+            temperature = None
 
         try:
-            current_consumption = "%.2f W" % \
```

Comment:
Why did you remove the current_consumption section?

Reply:
Since `current_power_watt` now is correctly renamed to `current_power_w` the current consumption will automatically be added as an attribute.